pyuvsim/utils.py

```diff
@@ -175,19 +175,32 @@ def check_file_exists_and_increment(filepath, extension=None):
     return filepath
 
 
-def write_uvdata(uv_obj, param_dict, return_filename=False, dryrun=False, out_format=None):
+def write_uvdata(
+    uv_obj,
+    param_dict,
+    return_filename=False,
+    dryrun=False,
+    out_format=None,
+    clobber=True,
+):
     """
     Parse output file information from parameters and write uvfits to file.
 
-    Args:
-        uv_obj: UVData object to write out.
-        param_dict: parameter dictionary defining output path, filename, and
-                    whether or not to clobber.
-        return_filename: (Default false) Return the file path
-        dryrun: (Default false) Don't write to file.
-        out_format: (Default uvfits) Write as uvfits/miriad/uvh5
+    Parameters
+    ----------
+    uv_obj : UVData Object
+        The object to be written out.
+    param_dict : Dict
+        parameter dictionary defining output path, filename, and whether or not to clobber.
+    return_filename : Bool
+        (Default false) Return the file path
+    dryrun : Bool
+        (Default false) Don't write to file.
+    out_format : Str
+        (Default uvfits) Write as uvfits/miriad/uvh5
```

Comment:
I'm confused -- should a user control file clobbering using the `clobber` keyword or using the param_dict? The clobber keyword is not listed in the docstring Parameters section, but the description of the param_dict suggests that's the way to control it.

Reply:
I have no idea honestly. Looks like it is picked out of the param dict in line 238. So presumably somewhere along the way the user gets to control it?

Reply:
But you added a `clobber` keyword to the function as well that is not documented in the docstring. Does it work in addition to the param_dict?

Reply:
oh! that's a mistake! thanks for catching. It is part of the param dict. Looks like I missed that, added it, then found it as part of the param dict and never removed that line